find_me_a_sandwich/app/models/favorite.rb

```diff
@@ -0,0 +1,4 @@
+class Favorite < ActiveRecord::Base
+  belongs_to(:user)
+  has_one(:menu_item)
```

Comment:
This should also be a `#belongs_to`.  `#belongs_to` is always on the model with the foreign key.